tests/framework/extensions/clusters/clusters.go

```diff
@@ -122,6 +123,63 @@ func CheckServiceAccountTokenSecret(client *rancher.Client, clusterName string)
 	}
 }
 
+// CreateRancherBaselinePSACT creates custom PSACT called rancher-baseline which sets each PSS to baseline.
+func CreateRancherBaselinePSACT(client *rancher.Client, psact string) error {
+	_, err := client.Steve.SteveType(PodSecurityAdmissionSteveResoureType).ByID(psact)
+	if err == nil {
+		return nil
+	}
+
+	template := &v3.PodSecurityAdmissionConfigurationTemplate{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: psact,
+		},
+		Description: "This is a custom baseline Pod Security Admission Configuration Template. " +
+			"It defines a minimally restrictive policy which prevents known privilege escalations. " +
+			"This policy contains namespace level exemptions for Rancher components.",
+		Configuration: v3.PodSecurityAdmissionConfigurationTemplateSpec{
+			Defaults: v3.PodSecurityAdmissionConfigurationTemplateDefaults{
+				Enforce: "baseline",
+				Audit:   "baseline",
+				Warn:    "baseline",
```

Comment:
it would make sense to me to have this defined in a const

Reply:
Sure, I can do that